src/main/java/de/tum/in/www1/exerciseapp/web/rest/FeedbackResource.java

```diff
@@ -54,10 +57,12 @@ public FeedbackResource(FeedbackRepository feedbackRepository, FeedbackService f
         if (feedback.getId() != null) {
             return ResponseEntity.badRequest().headers(HeaderUtil.createFailureAlert(ENTITY_NAME, "idexists", "A new feedback cannot already have an ID")).body(null);
         }
-        Feedback result = feedbackRepository.save(feedback);
-        return ResponseEntity.created(new URI("/api/feedbacks/" + result.getId()))
-            .headers(HeaderUtil.createEntityCreationAlert(ENTITY_NAME, result.getId().toString()))
-            .body(result);
+        Feedback savedFeedback = feedbackRepository.save(feedback);
```

Comment:
please don't send each feedback in its own REST request. This is not efficient. Rather send all feedback objects together with the manual result once

Reply:
Didn't occur to me originally. Changed to one POST request to /api/results.